packages/preferences/src/browser/style/preference-file.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .theia-settings-container .preference-file-button {
-  position: absolute;
-  padding: 2px 9px;
-  right: 4px;
+  margin-left: 0px;
+  padding-bottom: 6px;
+  padding-top: 5px;
```

Comment:
I can't seem to get the "Windows Exec" preference to show up when running in Firefox (with `yarn browser start`); I previously tested with Electron. How are you doing it?

Using exact pixel values seems like it would lead to more problems later (unless the height of the text field is also set in pixels, which it doesn't seem to be). I don't see any obvious way to make it automatically use the exact same amount of vertical space/all available vertical space (`height: 100%` didn't do it).

Reply:
I've just modified a preference to use the `typeDetails: { isFilepath: true }` property. See [here](https://github.com/eclipse-theia/theia/blob/63d66679f5a681a83affb7361333134d5d55f8cd/packages/external-terminal/src/electron-browser/external-terminal-preference.ts#L93). In my case, I've used the `window.title` preference.

> unless the height of the text field is also set in pixels, which it doesn't seem to be

It's not, but it's using `line-height` which is almost the same for the purpose of size calculation as plainly using `height`.

Reply:
I used `calc(var(--theia-content-line-height) + 4px)` (which is equivalent to 26px). That seems to give correct results with firefox and electron-chromium.

Reply:
@msujew Is the proposed solution fine with you? If so, we could get this PR merged.